validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -153,4 +153,24 @@ public void extra_config() {
     untilLogged(SYSTEM_CONFIG_APPLY, SYSTEM_CONFIG_APPLY_LEVEL);
   }
 
+  @Test(timeout = ONE_MINUTES_MS)
+  @Feature(stage = BETA, bucket = SYSTEM)
+  @Summary("Check that the device publishes minimum required log entries when receiving config")
+  public void config_logging() {
+    deviceConfig.system.min_loglevel = Level.DEBUG.value();
+    updateConfig("send device config set loglevel to debug");
+
+    // Wait for device to receive the config
+    // We can untilLogged the system.config.apply event, but following the natural ordering
+    // of receive --> parse --> apply may be more helpful
+    safeSleep(CONFIG_THRESHOLD_SEC * 2000);
+
+    deviceConfig.system.min_loglevel = Level.TRACE.value(); // Force a new config to be sent
+    updateConfig("device receive config");
```

Comment:
"loglevel to trace"

Reply:
there's also a new function I created, something like forceUpdate() -- does the same thing without arbitrarily changing a value...

Reply:
I tried it - but it doesn't seem to do the trick? 
Using force update - https://github.com/noursaidi/udmi/actions/runs/5740987796/job/15560832585#step:6:468
Setting loglevel to trace to trigger an update - https://github.com/noursaidi/udmi/actions/runs/5740097044/job/15558012902#step:6:518